src/main/resources/templates/laDigitalAssister/signature.html

```diff
@@ -1,22 +1,24 @@
 <!DOCTYPE html>
 <html th:lang="${#locale.language}">
-<head th:replace="fragments/head :: head(title='Scaffold')"></head>
+<head th:replace="fragments/head :: head(title=#{signature-title})"></head>
 <body>
 <div class="page-wrapper">
   <div th:replace="fragments/toolbar :: toolbar"></div>
   <section class="slab">
     <div class="grid">
       <div th:replace="fragments/goBack :: goBackLink"></div>
       <main id="content" role="main" class="form-card spacing-above-35">
-        <!--        <th:block-->
-        <!--            th:replace="'fragments/cardHeader' :: cardHeader(header=#{}, subtext=#{})"/>-->
-        <th:block th:replace="'fragments/form' :: form(action=${formAction}, content=~{::formContent})">
-          <th:block th:ref="formContent">
+                <th:block
+                    th:replace="'fragments/cardHeader' :: cardHeader(header=#{signature-header})"/>
+        <th:block th:replace="'fragments/form' :: form(action=${formAction}, content=~{::submitForm})">
```

Comment:
(Set submitted_at on submission)
```suggestion
        <th:block th:replace="'fragments/form' :: form(action=('/flow/' + ${flow} + '/'+ ${screen} + '/submit' ), content=~{::submitForm})">
```